### ISSUE TITLE
bug: custom LLM

### ISSUE THREAD

**Issue by l4b4r4b4b4** [langfuse/langfuse#1582]
### Describe the bug

Custom model name is not picked up.
WARNING:langfuse:Langfuse was not able to parse the LLM model. The LLM call will be recorded without model name. Please create an issue so we can fix your integration: https://github.com/langfuse/langfuse/issues/new/choose

### To reproduce

Setup vLLM openAI endpoint and langchain vLLMServe component. 
Set served model name in vllm, model name in LangChain llm component and setup new model in LangFuse with the same name.

**Comment by sir3mat** [comment]
Any news about this issue?

**Comment by l4b4r4b4b4** [comment]
yep it works. @hassiebp But I only tried with using default tokenizer (Claude). Can I define a model path to use the models specific tokenizer now?

**Comment by l4b4r4b4b4** [comment]
> @l4b4r4b4b4 Not sure I understand. Could you please explain what behavior you'd like to see?

For example: If I am using a llama-3 model that is being traced with langfuse I would like to use the llama-3 tokenizer for token count calculations.

**Comment by hassiebp** [comment]
Got it @l4b4r4b4b4 - we are currently supporting only Claude and OpenAI tokenizers. Please create a feature request if you'd like to see more supported!
